mlfinlab/data_structures/run_data_structures.py

```diff
@@ -197,11 +197,10 @@ def _get_expected_imbalance(self, window, imbalance_array):
                 imbalance_array['buy'][-ewma_window:], dtype=float)
             sell_sample = np.array(
                 imbalance_array['sell'][-ewma_window:], dtype=float)
-            buy_and_sell_imb = sum(buy_sample) + sum(sell_sample)
             exp_buy_proportion = ewma(
```

Comment:
this expression doesn't give is proportion, it gives expected buy imbalance instead